### ISSUE TITLE
Variant Description

### ISSUE THREAD

**Issue by empty** [empty/pestle-and-press#1]
Add variant descriptions that will swap out when you select the specified variant.

**Comment by empty** [comment]
@peverson16 Got a version of this working. I need to test it a bit more which I will do tomorrow but probably going to be tomorrow afternoon.

**Comment by empty** [comment]
@peverson16 This is an example of the format. You can go ahead and add them.
```
<div class="description" id="19197137649764">
<p>5mg of CBD<br />Contains less than .3% THC (FFB, 2014)</p>
<h4>Benefits</h4>
<p>Soothe Lip Balm with CBD has been designed to heal dry chapped skin, irritations from burns, abrasions, cuts and bites. Created using organic peppermint for its amazing adaptogenic qualities _ cooling and warming, calming and stimulating, along with relaxing and inspiring.</p>
<h4>Ingredients</h4>
<p>butyrospermum parkii (shea) butter**, theobroma cacao (cocoa) seed butter*, virgin cocos nucifera (coconut) oil*, beeswax, mentha piperita (peppermint) oil*, whole flower cbd, non-gmo tocopherol (vitamin e)<br /><br />*certified organic **fair trade</p>
<h4>Directions</h4>
<p>Suggested Use: Apply to dry, irritated skin.</p>
</div>
<div class="description" id="19197137715300">
<p>5mg of CBD<br />Contains less than .3% THC (FFB, 2014)</p>
<h4>Benefits</h4>
<p>Soothe Lip Balm with CBD has been designed to heal dry chapped skin, irritations from burns, abrasions, cuts and bites. Created using organic ginger, cardamom, and lime _ restorative, pain-relieving, and antiseptic.</p>
<h4>Ingredients</h4>
<p>butyrospermum parkii (shea) butter**, theobroma cacao (cocoa) seed butter*, virgin cocos nucifera (coconut) oil*, beeswax, zingiber officinale (ginger) oil*, elettaria cardamomum (cardamom) oil*, citrus aurantifolia (lime) oil*, whole flower cbd, non-gmo tocopherol (vitamin e)<br /><br />*certified organic **fair trade</p>
<h4>Directions</h4>
<p>Suggested Use: Apply to dry, irritated skin.</p>
</div>
```

**Comment by peverson16** [comment]
@empty could you check me on this.  when i go to the bath bombs page the selected option is "hypnosis" but its give me all four descriptions in a row.  When select a different variant it displays correctly. then when i go back to the "hypnosis" it is correct.  

below is what I have.

<div class="description" id="19197153509476">
<p>The Hypnosis bath bomb is the perfect before-bed bath bomb. Handcrafted with pure lavender essential oils, which has been used for thousands of years as a natural sleep aid. Topped with an amethyst and fresh lavender buds. Amethyst is highly beneficial in treating insomnia, especially when caused by an over-active mind.</p>
<h4>Product Information</h4>
<p>6 oz bath bomb<br /> 50mg of CBD per bath bomb</p>
<h4>Ingredients</h4>
<p>50mg Pure Hemp-Derived CBD, Organic Coconut Oil, Sodium Bicarbonate, Naturally-derived Citric Acid, Corn Starch, Epsom Salt, Organic Herbs, Essential Oils, Plant-based Color, Witch Hazel, Cleansed &amp; Charged Crystal.</p>
</div>
<div class="description" id="19197153542244">
<p>Named after Steve Nicks' famous "Dreams", this bomb is light and relaxing. Created with a blend of floral essential oils, a small smokey quartz, fresh chrysanthemum flowers, and lavender buds.</p>
<h4>Product Information</h4>
<p>6 oz bath bomb<br /> 50mg of CBD per bath bomb</p>
<h4>Ingredients</h4>
<p>50mg Pure Hemp-Derived CBD, Organic Coconut Oil, Sodium Bicarbonate, Naturally-derived Citric Acid, Corn Starch, Epsom Salt, Organic Herbs, Essential Oils, Plant-based Color, Witch Hazel, Cleansed &amp; Charged Crystal.</p>
</div>
<div class="description" id="19197153607780">
<p>The Aphrodite bath bomb is Handcrafted with all-natural aphrodisiac essential oils, making it the perfect couple or self love treat. Made with organic, lab-tested plant extract to ensure optimal pain relief and relaxation. Includes a cleansed, charged rose quartz stone to enhance positive and pure energies of deep love. Topped with rose buds and hibiscus petals.</p>
<h4>Product Information</h4>
<p>6 oz bath bomb<br /> 50mg of CBD per bath bomb</p>
<h4>Ingredients</h4>
<p>50mg Pure Hemp-Derived CBD, Organic Coconut Oil, Sodium Bicarbonate, Naturally-derived Citric Acid, Corn Starch, Epsom Salt, Organic Herbs, Essential Oils, Plant-based Color, Witch Hazel, Cleansed &amp; Charged Crystal.</p>
</div>
<div class="description" id="19197153640548">
<p>Handcrafted with organic, nourishing coconut oil, aphrodisiacal therapeutic-grade essential oils, organic rose buds and a cleansed and charged Rose Quartz - the stone of unconditional love. Whether sharing with a loved one or enjoying by yourself this bomb is the perfect self-care pick me up.</p>
<h4>Product Information</h4>
<p>6 oz bath bomb<br /> 50mg of CBD per bath bomb</p>
<h4>Ingredients</h4>
<p>Organic Hemp Oil, Organic Coconut Oil, Sodium Bicarbonate, Naturally-derived Citric Acid, Corn Starch, Epsom Salt, Organic Herbs, Essential Oils, Plant-based Color, Witch Hazel, Cleansed &amp; Charged Crystal.</p>
</div>

**Comment by empty** [comment]
@peverson16 Yes everything looks good. The initial load is still the piece I'm working out.

**Comment by peverson16** [comment]
@empty gotcha, yeah thats the only things thats not working. This is awesome though, so cool to be putting these in.  just finished nuleaf too. just a few more left.

**Comment by empty** [comment]
@peverson16 This should all be good now
